OpenSim/Examples/ExampleHopperDevice/helperMethods.h

```diff
@@ -184,7 +195,7 @@ inline void simulate(Model& model, SimTK::State& state, bool saveStatesFile)
 
     // Simulate until the user presses ESC (or enters 'q' if visualization has
     // been disabled).
-    while (true) {
+    while (!simulateOnce) {
```

Comment:
> Hm, perhaps we need to revise point 3 in CONTRIBUTING.md under "Making a Pull Request (PR)" to indicate that tests must pass and the developer must ensure that the code executes as designed?

Sounds good to me. I think that point 3 is really about making sure *other* / *all* tests pass (tests aside from the ones related to the feature being worked on). But I do think it would be good to add such a statement to that document.